portfolio/src/main/webapp/assets/js/script.js

```diff
@@ -99,4 +100,72 @@ function toggleImageGallery() {
     toggleButton.innerHTML = "view image gallery"
 
   }
+}
+
+function getComments() {
+    fetch("/data").then(response => response.json()).then((comments) => {
+        console.log(comments);
```

Comment:
I usually will remove these kind of statements for committed code, and only use for debugging :)